v_2_0_1/cloudconfig.go

```diff
@@ -0,0 +1,63 @@
+package v_2_0_1
```

Comment:
This is not just copy/paste. You edited the files and at this point nobody knows what you changed elsewhere. The copy PR is just about `cp src dst`. Nothing else. Then you can just merge right away like vendor PRs. Package name changes and template changes should then be done in the follow up PR.

Reply:
I'd be ok with package name change only. That declutters changes diff a little bit.

Reply:
Just saying, when you state a PR to be copy paste but it is not only copy paste trust is gone.